ci-operator/step-registry/baremetal/lab/install/agent/baremetal-lab-install-agent-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+RELEASE_IMAGE=$(oc adm release info --registry-config "${CLUSTER_PROFILE_DIR}/pull-secret" "${RELEASE_IMAGE_LATEST}" -o json|jq -r '.metadata.version')
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=INFO
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+ssh "${SSHOPTS[@]}" root@openshift-qe-055.arm.eng.rdu2.redhat.com <<EOF
```

Comment:
Maybe we would get some infosec alert email for the dns ?

Reply:
Acked, addressed this in an ENV